examples/platforms/efr32mg12/sleepy-demo/README.md

```diff
@@ -1,6 +1,6 @@
 # EFR32MG12 Sleepy Demo Example
```

Comment:
I see there are different changes to the efr32mg12, efr32mg13, and efr32mg21 versions of the file which indicates to me the contents are different.  Why are they different?  Shouldn't they all be the same?

Reply:
It looks like the `sleepy-demo-mtd` main.c file has been augmented to automatically wake the device every 5 seconds by setting `sButtonPressed` to true when 5000ms has elapsed on some timer. As for the motivation, the only thing I can think of is it makes the testing process contactless so you don't have to press a button. I don't know.

Looks like Marven Gilhespie from the High Wycombe office made the change in Jan 2020 https://github.com/openthread/openthread/commit/73e30d1d7b1454e0d82607381b6b9c1f79ff4cec

Reply:
Hi Bob - I was expecting the behavior to be the same too, but looks like, it is not. Based on the read-me, it appears that the button press functionality. has been incorporated differently for the 12,13 vs the 21 part